automation/Clean.ps1

```diff
@@ -0,0 +1,51 @@
+
+param(
+    [string]$SwitchName = "Default Switch",
+
+    [Parameter(Mandatory=$true)]
+    [string]$VMName,
+
+    [Parameter()]
+    [string]$Platform = "Hyper-V"
+)
+
+if ($VMName) {
+    try{
+        if ($Platform -eq "Hyper-V") {
+            $dirPath = "C:\Users\$env:USERNAME\.minikube\machines\$VMName"
+            Stop-VM -Name $VMName -Force
+            Remove-Vm -Name $VMName -Force
+        }
+        elseif ($Platform -eq "VirtualBox") {
+            $dirPath = "C:\Users\$env:USERNAME\VirtualBox VMs\$VMName"
+            & VBoxManage controlvm $VMName poweroff --type headless
+            & VBoxManage unregistervm $VMName --delete
+        }
```

Comment:
Log message for success and if the plaform is not supported

Reply:
I'll add a validated set to this file too